dev/io.openliberty.mail.2.0.internal/resources/io/openliberty/mail/resources/MailMessages.nlsprops

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME io.openlioberty.jakarta.mail.repository/resources/io.openliberty/jakarta/mail/repository/internal/resources/MailMessages.nlsprops
+#COMPONENTPREFIX CWWKX
+#COMPONENTNAMEFOR WebSphere Jakarta Mail API
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# Message prefix block: CWWKJ0200 - CWWKJ0299
+
+#-----  Jakarta Mail messages
+CWWKX0958I=CWWKX0958I: A mailSession object in the server.xml file successfully created a jakarta.mail.Session object.
```

Comment:
This message prefix does not match the block described above? Which one is correct?

Reply:
I attempted to use the block above but when adding a message that started with CWWKJ I received an error that that prefix was not supported. The doc for liberty states that the CWWKJ should be allowed.  So I opted to increment the value.